README.md

```diff
@@ -40,6 +40,39 @@ The image being pushed can again be modified using the environment variable:
 IMG=quay.io/user/hasoperator:next make docker-push
 ```
 
+## Deploying the Operator (non-KCP)
+
+The following section outlines the steps to deploy HAS on a physical Kubernetes cluster. If you are looking to deploy HAS on KCP, please see [this document](./docs/kcp.md).
+
+### Creating a GitHub Secret for HAS
+
+Before deploying the operator, you must ensure that a secret, `has-github-token`, exists in the namespace where HAS will be deployed. This secret must contain a key-value pair, where the key is `GITHUB_AUTH_TOKEN` and where the value points to a valid GitHub Personal Access Token.
+
+The token that is used here must have the following permissions set:
+- `repo`
+- `delete_repo`
+
+In addition to this, the GitHub token must be associated with an account that has write access to the GitHub organization you plan on using with HAS (see next section).
+
+For example, on OpenShift:
```

Comment:
should the key be `GITHUB_AUTH_TOKEN`

Reply:
The key should still be `token` for the secret, I'll fix that.

Reply:
Updated.